verifier-py/tests-integration/test_verifier.py

```diff
@@ -0,0 +1,36 @@
+import pathlib
+import subprocess
+
+import pytest
+
+
+PLAYBOOK_DIRECTORY = pathlib.Path(__file__).parents[2].absolute() / "data" / "playbooks"
+
+
+@pytest.mark.parametrize(
+    "filename",
+    [
+        "insights_remove.yml",
+    ],
+)
+def test_official_playbook(filename: str):
+    """Test playbook signed by Red Hat's GPG key.
+
+    In this test, the official playbooks are verified against the GPG key
+    the application ships.
+    """
+    playbook_content: str = (PLAYBOOK_DIRECTORY / filename).read_text()
+
+    result = subprocess.run(
+        ["python3", "-m", "insights_ansible_playbook_verifier"],
```

Comment:
hmm to be proper functional tests, the actual installed executable ought to be run here; this also means these functional tests cannot be run during a package build

Reply:
I'm aware of that, my plan was to have _something_ for now and change it later when we have proper testing setup.